package.json

```diff
@@ -23,6 +23,8 @@
     "htmlnano": "^0.1.6",
     "is-url": "^1.2.2",
     "js-yaml": "^3.10.0",
+    "lodash.get": "^4.4.2",
+    "lodash.omit": "^4.5.0",
```

Comment:
I'd rather avoid adding more dependencies if we can. omit could just be `delete config.plugins['postcss-modules']`, and get could be just be `config.plugins['postcss-modules']`.